py7zr/py7zr.py

```diff
@@ -971,20 +977,20 @@ def readall(self) -> Optional[Dict[str, IO[Any]]]:
         self._dict = {}
         return self._extract(path=None, return_dict=True)
 
-    def extractall(self, path: Optional[Any] = None, callback: Optional[ExtractCallback] = None) -> None:
+    def extractall(self, path: Optional[Any] = None, callback: Optional[ExtractCallback] = None, enable_symlink: bool = False) -> None:
```

Comment:
We should be True in default at first, then ask users to add their preference in their project.
We can change default to be False, after many users are aware of it.